pkg/cmd/backuprestore/backupserver.go

```diff
@@ -39,14 +37,17 @@ func NewBackupServer(ctx context.Context) *cobra.Command {
 			}
 		},
 	}
-	backupSrv.AddFlags(cmd.Flags())
+	backupSrv.AddFlags(cmd)
 	return cmd
 }
 
-func (b *backupServer) AddFlags(fs *pflag.FlagSet) {
+func (b *backupServer) AddFlags(cmd *cobra.Command) {
```

Comment:
why do we need to change to cobra here?

Reply:
so the `Prune` expects `*cobra.Command` , thats why

Reply:
then we can make that change when we introduce prune.Run

Reply:
👍🏽